src/constants/apiPreferredVersions.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+angular.module('openshiftCommonServices')
+  .constant('apiPreferredVersions', (function(undefined) {
```

Comment:
All caps since it's a constant?

I'd rather not start declaring functions as taking `undefined` since we don't do that anywhere else. Let's stick to the styles we've established when it makes sense for consistency in our code. Even if we don't have a README declaring the code style, we've established a certain style over time.

```js
.constant('API_PREFERRED_VERSIONS', function() {
  // ...
});
```

Reply:
oh def, sorry, copy paste oops (forgot if constants had any special treatment), def don't need that.